src/enclave/enclave_event_channel.c

```diff
@@ -145,18 +150,36 @@ void initialize_enclave_event_channel(
 
     evt_chn_lock = (struct ticketlock**)oe_calloc(
         evt_channel_num, sizeof(struct ticketlock*));
+    if (!evt_chn_lock)
+    {
+        sgxlkl_fail("Could not allocate memory for evt_chn_lock\n");
+    }
 
     vio_tasks =
         (struct lthread**)oe_calloc(evt_channel_num, sizeof(struct lthread*));
+    if (!vio_tasks)
+    {
+        sgxlkl_fail("Could not allocate memory for vio_tasks\n");
+    }
 
     _enc_dev_config = enc_dev_config;
     for (int i = 0; i < evt_channel_num; i++)
     {
         evt_chn_lock[i] =
             (struct ticketlock*)oe_calloc(1, sizeof(struct ticketlock));
+        if (!evt_chn_lock[i])
+        {
+            sgxlkl_fail("Could not allocate memory for evt_chn_lock[%i]\n", i);
+        }
+
         memset(evt_chn_lock[i], 0, sizeof(struct ticketlock));
```

Comment:
This is already `calloc`'d, I don't think the `memset` is necessary.

Reply:
Done.